.github/workflows/ttnn-run-sweeps-new.yaml

```diff
@@ -0,0 +1,49 @@
+name: "ttnn - Run sweeps new"
+
+on:
+  # schedule:
+  #   - cron: "0 1,7,13,19 * * *"
+  workflow_dispatch:
+  workflow_call:
+  push:
+    branch:
+      - jdesousa/sweeps-ci
+
+jobs:
+  build-artifact:
+    uses: ./.github/workflows/build-artifact.yaml
+    secrets: inherit
+
+  ttnn-sweeps:
+    needs: build-artifact
+    strategy:
+      # Do not fail-fast because we need to ensure all tests go to completion
+      # so we try not to get hanging machines
+      fail-fast: false
+      matrix:
+        arch: [grayskull, wormhole_b0]
+    env:
+      TT_METAL_ENV: ${{ vars.TT_METAL_ENV }}
+      ARCH_NAME: ${{ matrix.arch }}
+      ELASTIC_USERNAME: ${{ secrets.SWEEPS_ELASTIC_USERNAME }}
+      ELASTIC_PASSWORD: ${{ secrets.SWEEPS_ELASTIC_PASSWORD }}
+    environment: dev
```

Comment:
@tt-rkim is this environment: dev still in use?

Reply:
No, please rid of it

Reply:
Is there a diff environment? Or just remove it completely?